tests/test_api.py

```diff
@@ -24,16 +24,32 @@ def test_phone_registration_sends_message(client, mocker):
     phone_number = PHONE_NUMBER
     data = {"phone_number": phone_number}
     twilio_api = mocker.patch(
-        "phone_verify.services.PhoneVerificationService.send_verification"
+        "twilio.rest.Client"
     )
 
     response = client.post(url, data)
 
     assert response.status_code == 200
     assert twilio_api.called
     assert "session_token" in response.data
-    SMSVerification = apps.get_model("phone_verify", "SMSVerification")
-    assert SMSVerification.objects.get(
+    sms_verification = apps.get_model("phone_verify", "SMSVerification")
```

Comment:
I'd suggest something that can run the tests for all available backends. Do you have any suggestions where we can have DRY approach yet be able to test everything?